db/seeds/warning_templates/off-topic.md

```diff
@@ -0,0 +1,9 @@
+We've noticed that you've recently been writing a number of posts about topics that are beyond the scope of this site.
+
+You can find, what's on-topic and what's off-topic on $SiteName in the [help center](/help/faq).
+
+This is just a gentle reminder that we expect posts on this site to stay focused on the topic on-hand.
+We have a [Network of sites](https://codidact.com/) that you are free to use; you may find one of our other sites more suitable to some of your posts. If you would like to talk about the possibility of creating a site for a subject not currently covered, please write a post on [meta](https://meta.codidact.com/) with your request.
+Additionally, we have a $ChatLink available for more free-form discussion.
+
+While we appreciate your continued contributions within the scope of this site, we do ask that everyone make sure that the topic of their posts remain in scope.
```

Comment:
Let's not change from second person midway through: "...we do ask that you make sure that the topics of your posts remain in scope".